manifest-editor/components/tree/IIIFCanvas.tsx

```diff
@@ -48,11 +49,19 @@ const IIIFCanvas: React.FC<{ type: string; id: string; onClick: () => void }> =
             <ContainerColumn>
               {canvas &&
                 Object.entries(canvas).map(([key, value]) => {
-                  if (typeof value === "string") {
+                  if (typeof value === "string" || typeof value === "number") {
                     return (
                       <KeyValuePairString
                         key={key}
-                        onClick={() => console.log("clicked", key)}
+                        onClick={() => {
+                          if (
+                            key === "height" ||
+                            key === "width" ||
+                            key === "duration"
+                          ) {
+                            editorContext?.changeSelectedProperty("canvas", 3);
```

Comment:
Could we pass `key` into `changeSelectedProperty` and let it decide which tab to change to?

Reply:
We definitely could but it would require changes to how TabPanel component works but for sure something to add to my todo list!